.github/workflows/multi-arch-build.yaml

```diff
@@ -62,26 +62,37 @@ jobs:
           echo "$VERSION_OUTPUT"
           echo ::set-output name=version_output::$VERSION_OUTPUT
 
-      # Generate image related info - names, labels
+      # Generate image related info - names, labels, check whether to push
       - name: Generate image information
         id: image_info
         run: |
-          # have special environment variable for image name.
+          # Have special environment variable for image name.
           # Names are different for upstream and stable Buildah images
           if [ "${{ matrix.source }}" == "upstream" ]; then
             # quay.io/buildah/upstream:master
-            echo ::set-output name=buildah_tag::"${{ env.BUILDAH_QUAY_REGISTRY}}/${{ matrix.source }}:master"
+            # Always push upstream image to buildah repository
+            echo ::set-output name=buildah_push::'true'
+            # Never push upstream image to containers repository
```

Comment:
I'm confused by the comment and why you wouldn't want to push?

Reply:
Based on initial list of multi-arch images, for upstream case image should be pushed to `quay.io/buildah/upstream:main`, but not to `quay.io/containers/buildah:master` (or main)